HealthcareSystem/Backend/Model/Pharmacies/SftpConfig.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Backend.Model.Pharmacies
+{
+    public class SftpConfig
+    {
+        public string Host { get; set; } = "192.168.1.3";
+        public int Port { get; set; } = 22;
+        public string Username { get; set; } = "tester";
+        public string Password { get; set; } = "password";
```

Comment:
Pull these values out into appconfig.json and load them in Startup (same as RabbitMQ settings are loaded in your project, or mail settings in the PatientWebApp).

Reply:
I completely forgot to get rid of these after I finished setting up SFTP. Just pushed a fix for it.

Reply:
Fixed in a6b683487e40f42a1f66bbea3d1bd1d87c6d26f7.